package/boot/uboot-mvebu/patches/143-arm-mvebu-Espressobin-Update-README.marvell-file.patch

```diff
@@ -0,0 +1,84 @@
+From d7492bef8cb0f701c88e61c22f3880da6f315bf4 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Pali=20Roh=C3=A1r?= <pali@kernel.org>
+Date: Thu, 27 Aug 2020 15:01:28 +0200
+Subject: [PATCH] arm: mvebu: Espressobin: Update README.marvell file
```

Comment:
No need to add documentation.

Reply:
Yeah, I can drop this one. It's just the full set I applied.

Reply:
I kept the patch, since a follow-up patch also touches the file.

Reply:
That's fine.